src/middleware/devtools.ts

```diff
@@ -27,21 +29,19 @@ type StoreSetStateWithAction<S> = S extends { getState: () => infer T }
 interface DevtoolsOptions {
   name?: string
   anonymousActionType?: string
-  serialize?: {
-    options:
-      | boolean
-      | {
-          date?: boolean
-          regex?: boolean
-          undefined?: boolean
-          nan?: boolean
-          infinity?: boolean
-          error?: boolean
-          symbol?: boolean
-          map?: boolean
-          set?: boolean
-        }
-  }
+  serialize?:
+    | boolean
+    | {
+        date?: boolean
+        regex?: boolean
+        undefined?: boolean
+        nan?: boolean
+        infinity?: boolean
+        error?: boolean
+        symbol?: boolean
+        map?: boolean
+        set?: boolean
+      }
```

Comment:
So, this is breaking. Why did we have `options` in the first place?

Reply:
Not sure why there was the `options` thing but this is a good change. We can deprecate the existing shape and support both shaped old and new in 3.x

Reply:
nice idea. i'll try.

Reply:
#828